src/main/java/edu/kis/powp/jobs2d/features/DriverFeature.java

```diff
@@ -24,6 +28,10 @@ public static void setupDriverPlugin(Application application) {
         app = application;
         app.addComponentMenu(DriverFeature.class, "Drivers");
 
+        ApplyDriverDecoratorsSubscriber.getInstance().addDriverDecorator(usageMonitorFeature.getDeviceMonitorDriver());
```

Comment:
The dependency is still there! The goal was to remove any references to usage monitoring from this class and move them e.g. to UsageMonitorFeature.